packages/@react-spectrum/color/src/ColorArea.tsx

```diff
@@ -24,14 +25,15 @@ import {useProviderProps} from '@react-spectrum/provider';
 
 function ColorArea(props: SpectrumColorAreaProps, ref: FocusableRef<HTMLDivElement>) {
   props = useProviderProps(props);
+  let inputXRef = useRef(null);
+  let inputYRef = useRef(null);
+  let containerRef = useFocusableRef(ref, inputXRef);
+  [props, containerRef] = useContextProps(props, containerRef, ColorAreaContext);
```

Comment:
The existing RSP color components now consume from the RAC contexts even though I didn't rewrite them with RAC. That way they work inside a RAC ColorPicker.